pkg/operator/apiservicecontroller/apiservice_controller.go

```diff
@@ -230,3 +255,93 @@ func (c *APIServiceController) eventHandler() cache.ResourceEventHandler {
 		DeleteFunc: func(obj interface{}) { c.queue.Add(workQueueKey) },
 	}
 }
+
+// TODO: change the name
+func (c *APIServiceController) filterAPIServices() []*apiregistrationv1.APIService {
+
+	oauthOperatorPrecondition := c.oauthOperatorAvailableAndVersionMatch()
+
+	filteredApiServices := []*apiregistrationv1.APIService{}
```

Comment:
track this persistently on the struct so you can inform about deltas, not about the same things every time.

Reply:
> track this persistently on the struct so you can inform about deltas, not about the same things every time.

this still isn't persistent.